README.md

```diff
@@ -1,6 +1,7 @@
 # fueled plugin
 
 [![fastlane Plugin Badge](https://rawcdn.githack.com/fastlane/fastlane/master/fastlane/assets/plugin-badge.svg)](https://rubygems.org/gems/fastlane-plugin-fueled)
+![Version](https://img.shields.io/badge/version-0.2.6-blue.svg)
```

Comment:
Not so sure about this, shouldn't a plugin badge url be version independent. Let's publish releases from now on with tags and use this url instead

`https://img.shields.io/github/v/release/Fueled/fastlane-plugin-fueled`

Reply:
@humblerookie 
💯 you are right.